src/components/example/ECorousel.tsx

```
@@ -0,0 +1,35 @@
+import { Card, CardContent } from "@/components/ui/card"
```

ESLint

[error] Replace `"@/components/ui/card"` with `'@/components/ui/card'`

(prettier/prettier)